### ISSUE TITLE
used in production?

### ISSUE THREAD

**Issue by veeramarni** [pke/react-extension-point#2]
This seems to be a good idea, but wondering whether it was tested in production?

**Comment by pke** [comment]
Yes it is used in production successfully for some white-label mobile apps.
